### ISSUE TITLE
Questions about dataset partitioning

### ISSUE THREAD

**Issue by fengchexz** [tttgm/fellowshipai#1]
Is there any problem with the division of train dataset and test dataset? This method does not guarantee that all test set users are consistent with the training set, so collaborative filtering cannot be used for recommendation.
